controllers/rhmi/rhmi_controller.go

```diff
@@ -768,7 +785,7 @@ func (r *RHMIReconciler) processStage(installation *rhmiv1alpha1.RHMI, stage *St
 		if err != nil {
 			return rhmiv1alpha1.PhaseFailed, fmt.Errorf("could not create server client: %w", err)
 		}
-		product.Status, err = reconciler.Reconcile(context.TODO(), installation, &product, serverClient)
+		product.Status, err = reconciler.Reconcile(context.TODO(), installation, &product, serverClient, sku.GetProduct(productName))
```

Comment:
In general, I like this (passing in the specific product SKU config, rather than the whole SKU, but it requires considering how the reconciler will know if the SKU has just changed.